app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,131 @@
+.js-enabled .govuk-back-link {
+    visibility: visible;
+}
+
+.govuk-back-link {
+    visibility: hidden;
+}
```

Comment:
do we need this? is it not a play-frontend-hmrc component?

Reply:
It might be using the old version as there's no code coming through from play frontend